hotness/consumers.py

```diff
@@ -329,31 +324,29 @@ def _handle_anitya_update(self, upstream, package, msg):
 
             self.publish("update.bug.file", msg=dict(
                 trigger=msg, bug=dict(bug_id=bz.bug_id)))
+            _log.info("Filed Bugzilla #%i" % bz.bug_id)
 
             if is_monitored == 'nobuild':
                 _log.info("Monitor flag set to 'nobuild'.  "
                           "Skipping scratch build.")
                 return
 
-            if not self._rebase(package, version, upstream, bz):
-                # Fall back to the original process
-                _log.info("Now with #%i, time to do koji stuff" % bz.bug_id)
-                try:
-                    # Kick off a scratch build..
-                    task_id, patch_filename, description = self.buildsys.handle(
-                        package, upstream, version, bz)
-
-                    # Map that koji task_id to the bz ticket we want to pursue.
-                    self.new_triggered_task_ids[task_id] = dict(
-                        bz=bz, state=None, version=str(upstream))
-                    # Attach the patch to the ticket
-                    self.bugzilla.attach_patch(patch_filename, description, bz)
-                except Exception as e:
-                    heading = "Failed to kick off scratch build."
-                    note = heading + "\n\n" + str(e)
-                    _log.warning(heading)
-                    _log.warning(traceback.format_exc())
-                    self.bugzilla.follow_up(note, bz)
+            try:
+                _log.info('Starting scratch build for ' + str(package))
+                # Kick off a scratch build..
+                task_id, patch_filename, description = self.buildsys.handle(
+                    package, upstream, version, bz)
+
+                # Map that koji task_id to the bz ticket we want to pursue.
+                self.scratch_builds[task_id] = dict(
+                    bz=bz, state=None, version=str(upstream))
+                # Attach the patch to the ticket
+                self.bugzilla.attach_patch(patch_filename, description, bz)
+            except Exception as e:
+                _log.exception(e)
+                note = "An unexpected error occured creating the scratch build:"
+                note = note + "\n\n" + traceback.format_exc()
+                self.bugzilla.follow_up(note, bz)
```

Comment:
Sounds good. I've changed this to just point the user to the issue tracker and it logs the exception. A later PR can configure the logger to email anything at the exception level (although I think technically this is possible now since we expose a dictionary that's just handed to logging as a dictConfig).